pkg/odo/cli/component/update.go

```diff
@@ -44,6 +49,14 @@ var updateCmdExample = ktemplates.Examples(`  # Change the source code path of c
 	  %[1]s --binary ./downloads/sample.war
 		`)
 
+const deprecationWarning string = "WARNING: 'odo update' command will be removed in the future odo version.\n" +
+	"You should be using `odo config` command instead.\n" +
```

Comment:
I think we should add the same warning  in `odo update --help` 
Also can we change the color of message to yellow?

Reply:
![image](https://user-images.githubusercontent.com/7551036/87144588-a02ef800-c2c5-11ea-95dd-3b5532fd6b7b.png)


changes made.